templates/sb-development.ini.erb

```diff
@@ -3,13 +3,16 @@
 
 # defaults to 'False'
 debug = <%= @sideboard_debug_enabled %>
-priority_plugins = uber,
+priority_plugins = uber, panels
```

Comment:
This is unrelated to performance, but ``panels`` NEEDS to be first, because otherwise we might fail to load.  In practice this hasn't been happening for us, but it's totally happened to other people, especially when trying to get set up on Docker.